src/solver/optimisation/opt_gestion_des_bornes_cas_lineaire.cpp

```diff
@@ -177,10 +189,37 @@ void OPT_InitialiserLesBornesDesVariablesDuProblemeLineaire(PROBLEME_HEBDO* Prob
             Var = CorrespondanceVarNativesVarOptim->NumeroDeVariableDeLInterconnexion[Interco];
             CoutDeTransport = ProblemeHebdo->CoutDeTransport[Interco];
 
-            Xmax[Var] = ValeursDeNTC->ValeurDeNTCOrigineVersExtremite[Interco];
-
-            Xmin[Var] = -(ValeursDeNTC->ValeurDeNTCExtremiteVersOrigine[Interco]);
+            StartNodeAdequacyPatchType = ProblemeHebdo->StartAreaAdequacyPatchType[Interco];
+            EndNodeAdequacyPatchType = ProblemeHebdo->EndAreaAdequacyPatchType[Interco];
+            SetToZeroLinkNTCForAdequacyPatchFirstStep
+              = SetNTCForAdequacyFirstStep(AdequacyFirstStep,
+                                           StartNodeAdequacyPatchType,
+                                           EndNodeAdequacyPatchType,
+                                           SetToZero12LinksForAdequacyPatch,
+                                           SetToZero11LinksForAdequacyPatch);
```

Comment:
If we don't define them explicitly:
```suggestion
            SetToZeroLinkNTCForAdequacyPatchFirstStep
              = SetNTCForAdequacyFirstStep(AdequacyFirstStep,
                                           ProblemeHebdo->StartAreaAdequacyPatchType[Interco],
                                           ProblemeHebdo->EndAreaAdequacyPatchType[Interco],
                                           true,
                                           true);
```

Reply:
Done.